editor/src/components/editor/actions/actions.tsx

```diff
@@ -3490,16 +3490,26 @@ export const UPDATE_FNS = {
             jsxAttributesFromMap({
               alt: jsxAttributeValue('', emptyComments),
               src: imageAttribute,
-              style: jsxAttributeValue(
-                {
-                  position: 'absolute',
-                  left: relativeFrame.x,
-                  top: relativeFrame.y,
-                  width: relativeFrame.width,
-                  height: relativeFrame.height,
-                },
-                emptyComments,
-              ),
+              style: MetadataUtils.isFlexLayoutedContainer(
+                MetadataUtils.findElementByElementPath(editor.jsxMetadata, parent),
+              )
```

Comment:
We should probably extend this to check if we should be inserting with static positioning too (ideally using the same check as we do during reparenting)